apps/sms/views/shared/js/localization_helper.js

```diff
@@ -65,18 +73,22 @@
 
   exports.LocalizationHelper = {
     init: function () {
-      // This will be called during startup, and every time the language is
-      // changed
-      navigator.mozL10n.ready(function localized() {
```

Comment:
We don't need to wait for `ready` anymore because all `document.l10n` methods are safe to call right away.